### DISCUSSION TITLE
Adding a new emane model in rel4.8 core

### DISCUSSION THREAD

**Discussion by ullahhab** [coreemu/core#542] [Q&A]
Hi,
My name is Hamad and I have been working with Emane. As my development project assigned by AFRL, I modeled a new Emane model, and now AFRL wants us to integrate it into the core. I have been having some trouble bringing it to the Emane model in rel4.8. Could you guide me through the process, please?

Here is the heavyball.py that I wrote based on the documentation.
https://github.com/ullahhab/emane-ext-2-updated/blob/master/core-module/heavyball.py

**Comment by bharnden** [comment]
We are currently not focused on 4.8, considering how old it is at this point. I am also unfamiliar with the process at that point in time. I could only say to look at the other models at that time for reference.

If it were for recent CORE, you could attempt to follow this:
https://coreemu.github.io/core/emane.html#custom-emane-models
